stories/Atom/Images/Imagecaptioncredit/Imagecaption.jsx

```diff
@@ -18,8 +13,8 @@ export const Images = ({
   const cls = (...classes) => ((classes.filter(Boolean).length > 0) ? classes.filter(Boolean).join(' ') : null);
 
   return (
-    <figure className={cls('image-figcaption', `${size}`)}>
-      <div className='image-figcaption__cart scale-up'>
+    <figure className={cls('image-figcaption left-right', `${size}`)}>
+      <div className='image-figcaption__cart left-right scale-up'>
```

Comment:
@neelesh-tippe As discussed previously remove class `left-right` and handle this using `data-attribute`.